model/lollipop_keys.ts

```diff
@@ -0,0 +1,197 @@
+import * as t from "io-ts";
+import * as TE from "fp-ts/lib/TaskEither";
+import * as O from "fp-ts/lib/Option";
+
+import {
+  CosmosdbModelVersionedTTL,
+  RetrievedVersionedModelTTL
+} from "@pagopa/io-functions-commons/dist/src/utils/cosmosdb_model_versioned_ttl";
+import {
+  FiscalCode,
+  NonEmptyString,
+  PatternString
+} from "@pagopa/ts-commons/lib/strings";
+import { Container, RequestOptions } from "@azure/cosmos";
+import { NonNegativeInteger } from "@pagopa/ts-commons/lib/numbers";
+
+import { AssertionRef } from "@pagopa/io-functions-commons/dist/generated/definitions/lollipop/AssertionRef";
+import { AssertionType } from "@pagopa/io-functions-commons/dist/generated/definitions/lollipop/AssertionType";
+import { Timestamp } from "@pagopa/io-functions-commons/dist/generated/definitions/Timestamp";
+import {
+  CosmosErrors,
+  toCosmosErrorResponse
+} from "@pagopa/io-functions-commons/dist/src/utils/cosmosdb_model";
+import { flow, pipe } from "fp-ts/lib/function";
+import { enumType } from "@pagopa/ts-commons/lib/types";
+
+const LOLLIPOPKEYS_MODEL_PK_FIELD = "assertionRef" as const;
+const LOLLIPOPKEYS_MODEL_ID_FIELD = LOLLIPOPKEYS_MODEL_PK_FIELD;
```

Comment:
Can we differentiate model id and pk names?

Reply:
...forget about my comment, it's a versioned model, so model id will contain also the version. They are not the same, so we MUST differentiate.

Reply:
Nope. On versioned model the id field is always the id. Use only a value and the default to determine the other prop like in Profile https://github.com/pagopa/io-functions-commons/blob/master/src/models/profile.ts#L154